src/interp.cc

```diff
@@ -2249,6 +2249,36 @@ Result Thread::Run(int num_instructions) {
         break;
       }
 
+      case Opcode::I16X8Splat: {
+        uint16_t dLane = static_cast<uint16_t>(Pop<uint32_t>());
```

Comment:
nit: `lane`

Reply:
nit: `static_cast` is not needed since `Pop<T>` returns type `T` -- here and below

Reply:
Done.